cmd/kogito/command/flag/runtime_flag.go

```diff
@@ -38,8 +36,6 @@ type RuntimeFlags struct {
 // AddRuntimeFlags adds the RuntimeFlags to the given command
 func AddRuntimeFlags(command *cobra.Command, flags *RuntimeFlags) {
 	AddInstallFlags(command, &flags.InstallFlags)
-	AddInfinispanFlags(command, &flags.InfinispanFlags)
-	AddKafkaFlags(command, &flags.KafkaFlags)
 	AddMonitoringFlags(command, &flags.MonitoringFlags)
 	command.Flags().BoolVar(&flags.EnableIstio, "enable-istio", false, "Enable Istio integration by annotating the Kogito service pods with the right value for Istio controller to inject sidecars on it. Defaults to false")
 	command.Flags().BoolVar(&flags.EnablePersistence, "enable-persistence", false, "If set to true, deployed Kogito service will support integration with Infinispan server for persistence. Default to false")
```

Comment:
@vaibhavjainwiz I suppose that `enable-persistence` and `enable-events` can be removed from KogitoRuntime CLI flags, right?

Reply:
I think we can remove since he removed the Infinispan and Kafka creation. We won't remove that feature on this PR, but since it's done, yeah, we can get rid of these flags.